README.md

```diff
@@ -1,37 +1,44 @@
 # CNCF People Overview
 
-This repo stores the data that will populate the various people listings on cncf.io and control access to repositories in the CNCF GitHub org.
+This repo stores the data that
 
-People can update their listing by submitting a PR for approval. After a PR is merged, the CNCF site will reflect the update within 10 min. The listings are:
+1. populates various people listing pages on the cncf.io website
+2. grants access to repositories in the CNCF GitHub Org
+
+## Update your listing(s) on the cncf.io website pages with people.json
+
+You can update your listing in people.json by submitting a pull request for approval. After the PR is merged, the CNCF site will reflect the update within 10 min.
+
+The listings include:
 
 - [Ambassadors](https://www.cncf.io/people/ambassadors/)
 - [Governing Board](https://www.cncf.io/people/governing-board/)
 - [Staff](https://www.cncf.io/people/staff/)
 - [Technical Oversight Committee](https://www.cncf.io/people/technical-oversight-committee/)
 
-CNCF community members can request access to any repo by submitting a PR to change the config.yaml file in this repo.
+## Requesting access to CNCF repositories using CLOWarden and config.yaml
 
-## Listing Formats
+The CNCF has centralized access control for repos in this org using a application called CLOWarden [src](https://github.com/cncf/clowarden) [site](https://clowarden.io/audit/).
 
-### config.yaml configures CNCF org repository access
+Submitting Pull request to change config.yaml is how you request repo access.
```

Comment:
To request access to a repository, you can submit a pull request to modify the config.yaml file as described below.

Reply:
TY